tigerpath/majors_and_certificates/majors/CEE.json

```diff
@@ -7,11 +7,11 @@
   "min_needed": 1,
   "urls": [
     "https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering",
-    "http://www.princeton.edu/cee/forms/Yellow_Book_for_Class_of_2020_Mar.2017_3.7.pdf"
+    "https://cee.princeton.edu/undergraduate/paths-graduation"
   ],
   "contacts": [
     {
-      "type": "Departmental Representative",
```

Comment:
https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering

Reply:
This should be "Director of Undergraduate Studies"

Reply:
Fixed